### ISSUE TITLE
Changing the ID of a field won't change the for-attribute on the label

### ISSUE THREAD

**Issue by Dervisevic** [devcode-git/hosted-fields-sdk#9]
If you do change the ID for a field, the corresponding labels for-attribute will not be changed to match the ID of it's input. This results in breaking the click label to focus input functionality of browsers.

**Comment by ssolders** [comment]
Hi sorry for the delayed response. Good catch, should indeed we fixed. We'll include it in the upcoming version 1.2.0.

**Comment by ssolders** [comment]
Update, after looking into this a bit more closely the issue is located in another project with its own deploy-cycle.
Will fix it there and include it in its upcoming release.

It will be included in version 1.0.28 of Hosted Fields. When deployed, the fix will be available by specifying the `hostedfieldsurl`

Will let you know here when it's live.

```
// for test
hostedfieldsurl: 'https://test-hostedpages.paymentiq.io/1.0.28/index.html',

// for production
hostedfieldsurl: 'https://hostedpages.paymentiq.io/1.0.28/index.html',
```

**Comment by ssolders** [comment]
1.2.0 of this project has been deployed to NPM, [see changelog](https://github.com/devcode-git/hosted-fields-sdk/blob/master/CHANGES.md)

Hosted fields 1.0.28 has also been deployed, containing the fix for this issue. See earlier comment how to use this version.
